linode/helper/batch.go

```diff
@@ -11,35 +11,38 @@ type BatchFunction func() error
 // This is handy for running certain non-sequential API requests in parallel.
 func RunBatch(toExecute ...BatchFunction) error {
 	var wg sync.WaitGroup
+	wg.Add(len(toExecute))
```

Comment:
@zliang-akamai We have a ticket up to move this over to errgroups (TPT-2829), would it make sense to just do that as a part of this PR?